src/DBus.vala

```diff
@@ -110,6 +110,17 @@ public class Notifications.Server : Object {
         } else {
             var notification = new Notifications.Notification (app_name, app_icon, summary, body, actions, hints);
 
+            // Silence "Automatic suspend. Suspending soon because of inactivity." notifications
+            // These values and hints are taken from gnome-settings-daemon source code
+            if (hints.contains ("desktop-entry") && ((string) hints.get ("desktop-entry")) == "gnome-power-panel" &&
+                hints.contains ("x-gnome-privacy-scope") && ((string) hints.get ("x-gnome-privacy-scope")) == "system" &&
+                notification.priority == GLib.NotificationPriority.HIGH &&
+                app_icon == "" &&
```

Comment:
Can we also add a comment here about this app_icon part being important to make sure we don't block low-battery notifications? Just because it seems a little less obvious than the other stuff